config.go

```diff
@@ -84,7 +84,14 @@ func (p *parser) Parse(any *anypb.Any) (interface{}, error) {
 }
 
 func (p *parser) Merge(parent interface{}, child interface{}) interface{} {
-	panic("TODO")
+	parentConfig := parent.(*config)
+	childConfig := child.(*config)
+
+	newConfig := *parentConfig
+	if childConfig != nil {
+		newConfig = *childConfig
```

Comment:
better do merge logic, use the key values in child to override the newConfig.

Reply:
oh, I missed this one.

Reply:
Done.